docs/api/portable-stories-playwright.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Portable stories in Playwright'
+---
+
+export const SUPPORTED_RENDERERS = ['react', 'vue'];
+
+(⚠️ **Experimental**)
+
+<If notRenderer={SUPPORTED_RENDERERS}>
+
+<Callout variant="info">
+
+Portable stories are currently only supported in [React](?renderer=react) and [Vue](?renderer=vue) projects.
+
+</Callout>
+
+<!-- End non-supported renderers -->
+
+</If>
+
+<If renderer={SUPPORTED_RENDERERS}>
+
+Portable stories are Storybook [stories](../writing-stories/index.md) which can be used in external environments, such as [Playwright Component Tests (CT)](https://playwright.dev/docs/test-components).
+
+Normally, Storybok composes a story and its [annotations](#annotations) automatically, as part of the [story pipeline](#story-pipeline). When using stories in Playwright CT, you can use the [`createTest`](#createtest) function, which extends Playwright's test functionality to add a custom `mount` mechanism, to take care of the story pipeline for you.
+
+## createTest
+
+(⚠️ **Experimental**)
+
+Instead of using Playwright's own `test` function, you can use Storybook's special `createTest` function which will extend Playwright's test functionality to add a custom `mount` mechanism which will load, render, and play the story. This function is experimental and is subject to changes.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/portable-stories-playwright-ct.ts.mdx',
+    'vue/portable-stories-playwright-ct.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<Callout variant="warning">
+
+Please note the [limitations of importing stories in Playwright CT](#importing-stories-in-playwright-ct).
+
+</Callout>
+
+### Type
+
+```ts
+createTest(
+  baseTest: TK
+) => TK
```

Comment:
@yannbf — Not sure about the types here.

Reply:
Great question, these are types from the Playwright's test function, and the return type is also the type of a Playwright's test function, however with a different `mount` argument to that function. @JReinhold do you have any idea on how to present this here (and the TK's right below this comment) in a way that helps users?

Reply:
So the "correct" type of this would be

```ts
import type { TestType } from '@playwright/experimental-ct-core';

createTest(
  baseTest: TestType
) => TestType
```

But that seems a little verbose for these docs maybe?
I suggest we reference the [Fixture documentation from Playwright](https://playwright.dev/docs/test-fixtures#creating-a-fixture), because that is actually what we do here, we override the base Fixture.